### PR TITLE
Add Docker CE installation instructions page

### PR DESCRIPTION
**Pull request by hrismarin**
See: https://github.com/coreos/fedora-coreos-docs/issues/639

### CODE REVIEW

File: modules/ROOT/pages/docker-ce.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+= Installing Docker CE
+
+By default, Fedora CoreOS comes with out-of-the-box support for `docker` CLI (as provided via https://mobyproject.org/[Moby]). However, in some cases Docker Community Edition (CE) may be preferred for various reasons. This page explains how to replace the provided version with the latest from the upstream Docker sources.
```
Comment (travier) [inline modules/ROOT/pages/docker-ce.adoc:3]: Styling for the entire change: asciidoc works best with a single sentence per line, each sentence on it's own line.

File: modules/ROOT/pages/docker-ce.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+The recommended approach from the official https://docs.docker.com/engine/install/fedora/[Docker documentation] is to add the Docker repository to your system. You can then install and update Docker CE from this repository.
+
+
+== Installing Docker CE on a running system
+
+First you need to remove `moby-engine` and several other conflicting packages that ship by default in the Fedora CoreOS image. Next, download and setup the Docker repository. Afterward, you install the necessary packages, and finally reboot the system.
+
+[source, bash]
+----
+sudo rpm-ostree override remove moby-engine containerd runc
```
Comment (travier) [inline modules/ROOT/pages/docker-ce.adoc:14]: You can combine both rpm-ostree commands into a single one

File: modules/ROOT/pages/docker-ce.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+sudo rpm-ostree install --assumeyes --allow-inactive docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+sudo reboot
+----
+
+=== Upgrading Docker CE
+
+Docker CE should be upgraded automatically with each new release of Fedora CoreOS.
+
+
+== Installing Docker CE on first boot
```
Comment (travier) [inline modules/ROOT/pages/docker-ce.adoc:26]: We usually start with the Ignition config section first and then with the "how to update existing nodes" section.

**Review by travier — COMMENTED** [review]
Thanks for working on this. Added some minor comments.

**Review by travier — APPROVED** [review]
LGTM. Thanks

**Review by dustymabe — COMMENTED** [review]
this looks pretty good. Maybe we move it in the nav to under the OS Extensions entry since they are similar?

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**